test/src/test/java/jenkins/security/stapler/DoActionFilterTest.java

```diff
@@ -350,7 +350,7 @@ public void validateCrumb(StaplerRequest request, String submittedCrumb) {
         WebRequest settings = new WebRequest(new URL(j.getURL(), "testNewRulesOk/annotatedJavascriptScriptMethod/"));
         settings.setAdditionalHeader("Content-Type", "application/x-stapler-method-invocation");
         settings.setHttpMethod(HttpMethod.POST);
-        settings.setRequestBody(JSONArray.fromObject(Arrays.asList()).toString());
+        settings.setRequestBody(JSONArray.fromObject(Collections.emptyList()).toString());
```

Comment:
Not sure if this is really more readable than
```
settings.setRequestBody("[]");
```

Reply:
I would prefer this one as well.